carddav2fb.php

```diff
@@ -44,6 +44,7 @@
 $config['tmp_dir'] = sys_get_temp_dir();
 $config['fritzbox_ip'] = 'fritz.box';
 $config['fritzbox_ip_ftp'] = 'fritz.box';
+$config['fritzbox_force_local_config'] = false;
```

Comment:
Please rename this option to something like ```$config['fritzbox_force_local_login']``` as this better explains what this option is actually doing.